packages/zk-transfer-web/public/index.html

```diff
@@ -20,6 +20,15 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Zero Knowledge</title>
+    <script type="text/javascript">
+      window["RINKEBY.ETH_URL"] = "https://ganache.arcology.nyc:8547"
+      window["RINKEBY.DB_URL"] = "https://ganache.arcology.nyc:7001"
```

Comment:
It turns out adding these two lines gives us the wrong hostnames for the blockchain and the database.
```
 window["RINKEBY.ETH_URL"] = "https://ganache.arcology.nyc:8547"
      window["RINKEBY.DB_URL"] = "https://ganache.arcology.nyc:7001"
```
Can you please remove them, and try refreshing the page?
The value `demo.chainId` should be `4` for `RINKEBY`, and currently it would be `22` with the hosts set above.

Reply:
![Screenshot 2020-01-13 at 1 15 26 PM](https://user-images.githubusercontent.com/29642861/72255354-cfaac380-3606-11ea-9f26-926ac827beb1.png)
I think it changed. But we need am still getting an error.

Reply:
Sorry, you don't need the line with `RINKEBY.ETH_URL`,
but you do need the second line with `RINKEBY.DB_URL`. Try that and let me know if it doesn't work.

Reply:
It worked. :+1: